packages/components/src/font-size-picker/test/index.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * External dependencies
+ */
+import { render, fireEvent, screen } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import FontSizePicker from '../';
+
+describe( 'FontSizePicker', () => {
+	describe( 'onChange values', () => {
+		it( 'should not use units when the initial value is a number', () => {
```

Comment:
Should we check that the units input is not render or do you think it should be rendered in this case?

Reply:
Do you mean units "switcher"? It's is still rendered but has no effect since we only use number values.

I think toggling between `NumberControl` and `UnitControl` based on has units value will introduce more complexity to the component.

![CleanShot 2021-07-30 at 13 37 06](https://user-images.githubusercontent.com/240569/127634040-cc46b175-c3d2-425a-aec1-d59f62b320af.png)

Reply:
> I think toggling between NumberControl and UnitControl based on has units value will introduce more complexity to the component.

Can't we just pass an empty array of units or something like that in that case? It feels weird for a user to select "em" but have pixels applied.

Reply:
Done. It will now only display the "PX" unit label when there are no units.